core/shared/src/test/scala-3/me/shadaj/scalapy/py/SpecialSyntaxTest.scala

```diff
@@ -0,0 +1,5 @@
+package me.shadaj.scalapy.py
+
+import org.scalatest.funsuite.AnyFunSuite
+
+trait SpecialSyntaxTestScala2 extends AnyFunSuite {}
```

Comment:
Nit: no need for empty brackets

Reply:
I think we also don't need this trait in `scala-3` folder because we have something specific (bracket syntax) only for Scala 2 version at the moment. But in the future if we would like to test something specific in Scala 3 we can create `trait SpecialSyntaxTestScala3`

Reply:
Maybe we can rename it into something else like for a specific version. But I cannot think of a good one.

Reply:
Oh, at first I didn't understand it correctly. So, I was thinking that we could create another SpecialSyntaxTest class just for Scala 2 and Scala 3 versions and not to extend them (like separate test class). But if we want to put tests for specific version to the trait and extend it, then I think it would be better to rename this trait to make it more general (maybe for example `SpecificVersionSpecialSyntaxTest`). What do you think @shadaj ?